### DISCUSSION TITLE
Unable to open a port on security onion 2.4.30

### DISCUSSION THREAD

**Discussion by Wilkins00** [Security-Onion-Solutions/securityonion#12786] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

48

### Storage for /

100

### Storage for /nsm

3.5 tb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,

I currently having issue with the security onion 2.4.30 and open the port 5044 on the firewall to ingest the log of sysmon into the logstash , my current architecture is one manage and search node and another Heavy node, I check the manage and search and the port 5054 is listening for 0.0.0.0:5044 , however I am unable to open the port on manage and search node or heavy node either  , I have tried everything on the doc and nothing seem to works, please I would appreciate you help , I leave some screenshot below.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/20390864/3d7a4ff1-7d49-4c35-926d-2be2d2072839)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/20390864/d0b89fb9-df9f-4d24-99b4-0d648e0251ec)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/20390864/2dd4b418-5cfd-4fa0-9dcc-61218976ce2c)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/20390864/ac6fbd60-97b5-45cf-a103-d76d9b28e264)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/20390864/91eb874f-6e21-4ab7-8c21-19202446fc5f)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
All you should have to do is add the IP in the below section, there is already a portgroup for 5044 for a managersearch node.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63416267/d5f48b1d-75d8-4d8b-bc2a-fd7721a26e1d)

**Reply by Wilkins00** [reply]
Hello,
I have set up putting the beat_endpoint and all the corresponding field and it refuse to open the port.

**Reply by cm-ops** [reply]
Just so I am clear, `port 5044` is not open on the managersearch?

**Reply by Wilkins00** [reply]
Yes, is not open.

**Comment by InfosecGoon** [comment]
How are you sending the sysmon logs? You should be using the Elastic Agent installed on that Windows endpoint, in which case you would need to add the endpoints IP address to the elastic_agent_endpoint host group, not beats_endpoint.
